### DISCUSSION TITLE
What system is this for?

### DISCUSSION THREAD

**Discussion by GTEM19889** [lavv17/lftp#684] [Q&A]
I have read the readme, looked at the website but cannot figure out whether this is for Windows, Linux, or Apple system (all, or none)  
I am particularly interested in trying the mirroring abilities but need to find out what system this is for before trying it.

Might be an idea to make this clear on the website and the readme

**Comment by goyalyashpal** [comment]
I _think_ this is for linux. But msys2 on windows also has this (which is how i am using this).

Don't ask me about apple lol.

**Comment by GTEM19889** [comment]
Thank you for response, I have already forgotten about this as I managed to find another solution that works really well.

**Comment by goyalyashpal** [comment]
what solution is that? and what are/were ur requirements & use case? can u share?
